src/stratis_cli/_error_reporting.py

```diff
@@ -115,9 +116,18 @@ def _interpret_errors_0(error):
 
     # We have observed two causes of this problem. The first is that
     # stratisd is not running at all. The second is that stratisd has not
-    # yet established its D-Bus service.
+    # yet established its D-Bus service. While coverage of the first condition
+    # can be replicated by simply not starting stratisd, the second condition
+    # is difficult to replicate through testing.
     if error.get_dbus_name() == "org.freedesktop.DBus.Error.NameHasNoOwner":
-        return "Most likely stratis is unable to connect to the stratisd D-Bus service."
+        for proc in psutil.process_iter():
+            try:
+                if proc.name() == "stratisd":  # pragma: no cover
+                    return "Most likely stratis is unable to connect to the stratis D-Bus service."
+            except psutil.NoSuchProcess:  # pragma: no cover
+                pass
```

Comment:
Is this the best way to handle this exception?

Reply:
Yes, I think so.